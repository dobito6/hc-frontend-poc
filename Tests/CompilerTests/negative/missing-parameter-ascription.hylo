//! no-std

fun f(x) {}
