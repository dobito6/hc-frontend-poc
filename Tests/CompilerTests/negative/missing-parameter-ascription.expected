negative/missing-parameter-ascription.hylo:3.7-8: error: parameter requires ascription
fun f(x) {}
      ^
